contracts/src/periphery/cert/libraries/EigenDACertVerificationLib.sol

```diff
@@ -29,10 +29,12 @@ library EigenDACertVerificationLib {
     error InvalidInclusionProof(uint256 blobIndex, bytes32 blobHash, bytes32 rootHash);
 
     /// @notice Thrown when security assumptions are not met
-    /// @param gamma The difference between confirmation and adversary thresholds
-    /// @param n The calculated security parameter
-    /// @param minRequired The minimum required value for n
-    error SecurityAssumptionsNotMet(uint256 gamma, uint256 n, uint256 minRequired);
+    /// @param confirmationThreshold The confirmation threshold
+    /// @param adversaryThreshold The adversary threshold
+    /// @param reconstructionThreshold The reconstruction threshold
```

Comment:
I feel very strongly about and would really appreciate if you tried to write meaningful and useful comments. To me these comments are worse than not having comments. If you don't understand the params well enough to describe them, then its fine to tag someone else to provide those explanations. Let's all help each other make the documentation useful.

Reply:
Okay, then I'm tagging in @mooselumph because IDK at all really.